fragments/app-sso-client/accelerator.yaml

```diff
@@ -32,3 +32,38 @@ engine:
           options:
             key: "'$.spec'"
             yaml: "'serviceClaims:\n  - name: ' + #workloadName + '\n    ref:\n      apiVersion: services.apps.tanzu.vmware.com/v1alpha1\n      kind: ResourceClaim\n      name: ' + #workloadName + '-client-registration-claim'"
+    - include: [ "README.md" ]
+      chain:
+        - type: ReplaceText
+          regex:
+            pattern: "(?<existingContent>[\\s\\S]*)(?<endOfFile>\n)"
+            with: |
+              '${existingContent}
+              
+              > **_NOTE:_** Below you can find additional information concerning creation of AppSSO relevant resources. 
+
+              ## Resources needed for AppSSO integration
+              
+              Please first familiarize yourself with basic concepts around using AppSSO to secure your workload on TAP following 
+              ["Securing your first Workload" tutorial](https://docs.vmware.com/en/Application-Single-Sign-On-for-VMware-Tanzu/1.0/appsso/GUID-app-operators-tutorials-index.html)
+              .
```

Comment:
we have a new docs location inside the tap hierarchy for tap 1.3 - https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-index.html

Reply:
would that URL then become -  https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-index.html once docs are published?

Reply:
yes, I think so.

Reply:
I've changed the link, although the new one is not currently available.